RegTesting.Tests.Framework/Logic/BasePageObject.cs

```diff
@@ -41,14 +43,11 @@ public virtual void ApplySettings()
 
 		}
 
-		public virtual string CreatePageUrl(params string[] urlParameters)
+		public virtual string CreatePageUrlWithParameters(IEnumerable<string> urlParams)
 		{
-			//Override this method when the pageObject is not available at the root of your BaseUrl.
-			return String.Empty;
+			return null;
```

Comment:
This method, as it is used now, returns a relativ url.
So, if there is no override, it should return String.Empty to use the root in that case.